server/build.go

```diff
@@ -302,7 +302,7 @@ func (task *BuildTask) build(tracing *stringSet) (esm *ESM, err error) {
 					if task.BundleMode && !extraExternal.Has(specifier) {
 						pkgNameInfo := parsePkgPath(specifier)
 						if !builtInNodeModules[pkgNameInfo.Name] {
-							_, ok := npm.PeerDependencies[pkgNameInfo.Name]
+							_, ok := npm.PeerDependencies[pkgNameInfo.Fullname]
```

Comment:
I'm pretty sure deps are always declared with scopes

Reply:
yeah, you fixed it